server/src/main/java/io/druid/segment/realtime/plumber/RealtimePlumber.java

```diff
@@ -363,7 +363,9 @@ private Sink getSink(long timestamp)
                                           }
                                         }
                                     )
-                                )
+                                ),
+                                "query/wait/time",
+                                ImmutableMap.of("segment", theSink.getSegment().getIdentifier())
```

Comment:
@drcrallen i don't see where it was removed ? ServerManager [has it] (https://github.com/b-slim/druid/blob/59ea186af7a539cdf6a203abaf912649c7ef41f6/server/src/main/java/io/druid/server/coordination/ServerManager.java#L446-L446). What's wrong with high cardinality ?